uw-frame-static/index.html

```diff
@@ -15,11 +15,20 @@
   <link ng-href="css/themes/uw-{{portal.theme.name}}.css" rel="stylesheet" type="text/css"/>
   <link href="my-app/my-app.css" rel="stylesheet" type="text/css"/>
   <link rel="shortcut icon" href="bower_components/uw-ui-toolkit/dist/img/favicon.ico" type="image/x-icon"/>
+  <link rel="import" href="static.html" />
```

Comment:
That's why I check for `truthy` content. This will work with most, and browsers that don't support it just will load like it does today.